messaging/Controllers/SteveController.cs

```diff
@@ -21,12 +21,12 @@ public class SteveController : BundlesController
 
         protected override IQueryable<OutgoingMessageItem> ExcludeRetrieved(IQueryable<OutgoingMessageItem> source)
```

Comment:
can be deleted because the override is now the same as the base class's implementation